grml-debootstrap

```diff
@@ -1427,60 +1426,60 @@ prepare_vm() {
 
   ORIG_TARGET="$TARGET" # store for later reuse
 
+  case "$ARCH" in
+    i386|amd64) true;;
+    arm64)      VMEFI=1;;
+  esac
+
+  if [ "$ARCH" = 'arm64' ]; then
+    # arm64 support largely only exists for GPT
+    VMEFI=1
+  fi
+
   if [ -n "$VMFILE" ]; then
     qemu-img create -f raw "${TARGET}" "${VMSIZE}"
   fi
   if [ -n "$VMEFI" ]; then
     parted -s "${TARGET}" 'mklabel gpt'
     parted -s "${TARGET}" 'mkpart ESP fat32 1MiB 101MiB'
     parted -s "${TARGET}" 'set 1 boot on'
-    parted -s "${TARGET}" 'mkpart bios_grub 101MiB 102MiB'
-    parted -s "${TARGET}" 'set 2 bios_grub on'
-    parted -s "${TARGET}" 'mkpart primary ext4 102MiB 100%'
-
-  else
-    # arm64 support largely only exists for GPT
-    if [ "$ARCH" = 'arm64' ]; then
-      einfo "Setting up GPT partitions for arm64"
-      parted -s "${TARGET}" 'mklabel gpt'
-      parted -s "${TARGET}" 'mkpart ESP fat32 1MiB 10MiB'
-      parted -s "${TARGET}" 'set 1 boot on'
-      parted -s "${TARGET}" 'mkpart LINUX ext4 10MiB 100%'
+    if [ "$ARCH" = 'arm64' ] || [ "$ARCH" = 'i386' ]; then
+      # No need for a bios_grub parttion on ARM64 or i386
+      parted -s "${TARGET}" 'mkpart primary ext4 101MiB 100%'
```

Comment:
We disable bios_grub on i386 because we don't have according grub-cloud-i386 packages there, right?